source/agora/network/Client.d

```diff
@@ -624,12 +628,21 @@ public class NetworkClient
 
             foreach (index, conn; this.connections)
             {
+                scope (exit)
+                {
+                    if (this.connections_version > curr_conn_version)
+                    {
+                        curr_conn_version = this.connections_version;
+                        continue RETRY;
```

Comment:
We should not jump from a `scope (exit)`

Reply:
updated